config/routes.rb

```diff
@@ -1,6 +1,12 @@
+# config/routes.rb
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # List all posts for a given user
+  get '/users/:user_id/posts', to: 'posts#index', as: :user_posts
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  # Show a specific post
+  get '/posts/:id', to: 'posts#show', as: :post
+
+  # Create a new comment for a specific post
+  get '/posts/:post_id/comments/new', to: 'comments#new', as: :new_comment
+  post '/posts/:post_id/comments', to: 'comments#create', as: :comments
```

Comment:
- [ ] Great work on making the routes but i guess you missed some routes 😨 Kindly add routes to get all the users and specific user 👍 to follow the project requirements 😊
- [ ] It would be great if you make your routes as below to follow the new sytanx 👍

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```